pkg/model/components/containerd.go

```diff
@@ -49,7 +49,7 @@ func (b *ContainerdOptionsBuilder) BuildOptions(o interface{}) error {
 			if b.IsKubernetesGTE("1.23") {
 				containerd.Version = fi.PtrTo("1.6.20")
 				containerd.Runc = &kops.Runc{
-					Version: fi.PtrTo("1.1.4"),
+					Version: fi.PtrTo("1.1.7"),
```

Comment:
@heybronson Could you try setting the default to 1.1.5?
I think there's something broken with the latest releases of runc. We should probably create an issue with https://github.com/opencontainers/runc.

```suggestion
					Version: fi.PtrTo("1.1.5"),
```

Reply:
Will do, thanks for the response!